implants/lib/eldritch/src/pivot.rs

```diff
@@ -96,4 +103,105 @@ fn methods(builder: &mut MethodsBuilder) {
     //   ssh_copy_impl::ssh_copy(target, port, username, password, key, command, shell_path, src, dst)?;
     //   Ok(NoneType{})
     // }
-}
\ No newline at end of file
+}
+
+
+
+// SSH Client utils
+struct Client {}
+
+#[async_trait]
+impl client::Handler for Client {
+    type Error = russh::Error;
+
+    async fn check_server_key(
+        self,
+        _server_public_key: &key::PublicKey,
+    ) -> Result<(Self, bool), Self::Error> {
+        Ok((self, true))
+    }
+}
+
+pub struct Session {
+    session: client::Handle<Client>,
+}
+
+impl Session {
+    async fn connect(
+        user: String,
+        password: Option<String>,
+        key: Option<String>,
+        key_password: Option<&str>,
+        addrs: String,
+    ) -> anyhow::Result<Self> {
+        let config = client::Config {
+            ..<_>::default()
```

Comment:
wut on God's green earth is going on here?

Reply:
This is I believe similar to the javascript "the rest" notation `...` and is how you can tell a rust type to populate the rest of a structs fields with the defaults.